nodejs/monitoring.js

```diff
@@ -1,18 +1,30 @@
-const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
-const { OTLPMetricExporter } =  require('@opentelemetry/exporter-metrics-otlp-grpc');
+const { MeterProvider, PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
+const { OTLPMetricExporter } =  require('@opentelemetry/exporter-metrics-otlp-proto');
 
 const collectorOptions = {
   // url is optional and can be omitted - default is grpc://localhost:4317
   url: 'grpc://<IP of signoz backend>:4317',
+  // concurrencyLimit: 1, // an optional limit on pending requests
+
+  // k8s urls might be
+
+  // grpc://${HELM_RELEASE}-otel-collector.${SIGNOZ_NAMESPACE}.svc.cluster.local:4317"
+  // url: "grpc://my-release-otel-collector.platform.svc.cluster.local:4317"
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options

> Additionally, the endpoint MUST accept a URL with a scheme of either http or https

We need to check if the scheme has to be `grpc` because the spec says it should be `http` or `https`